openshift.spec

```diff
@@ -69,37 +96,72 @@ do
 done
 
 install -d -m 0755 %{buildroot}%{_unitdir}
-install -m 0644 -t %{buildroot}%{_unitdir} rel-eng/openshift.service
+install -m 0644 -t %{buildroot}%{_unitdir} rel-eng/openshift-master.service
+install -m 0644 -t %{buildroot}%{_unitdir} rel-eng/openshift-node.service
 
 mkdir -p %{buildroot}%{_sysconfdir}/sysconfig
-install -m 0644 rel-eng/openshift.sysconfig %{buildroot}%{_sysconfdir}/sysconfig/openshift
-
-mkdir -p %{buildroot}/var/log/%{name}
+install -m 0644 rel-eng/openshift-master.sysconfig %{buildroot}%{_sysconfdir}/sysconfig/openshift-master
+install -m 0644 rel-eng/openshift-node.sysconfig %{buildroot}%{_sysconfdir}/sysconfig/openshift-node
 
 mkdir -p %{buildroot}%{_sharedstatedir}/%{name}
 
 ln -s %{_bindir}/openshift %{buildroot}%{_bindir}/osc
 
+mkdir -p %{buildroot}/usr/lib/tuned/openshift-node
+install -m 0644 -t %{buildroot}/usr/lib/tuned/openshift-node tuned/openshift-node/tuned.conf
+
 %files
 %defattr(-,root,root,-)
 %doc README.md LICENSE
-%dir /var/log/%{name}
 %{_bindir}/openshift
 %{_bindir}/osc
-%{_unitdir}/*.service
-%config(noreplace) %{_sysconfdir}/sysconfig/openshift
 %{_sharedstatedir}/%{name}
```

Comment:
If we're calling it /var/lib/openshift in the file then I'd say eschew the macro... it just confuses me :)